website/docs/addon_aquarium_developer.md

```diff
@@ -0,0 +1,168 @@
+---
+id: addon_aquarium_developer
+title: Aquarium
+sidebar_label: Aquarium
+---
+
+import ReactMarkdown from "react-markdown";
+import versions from '@site/docs/assets/json/Ayon_addons_version.json'
+
+<ReactMarkdown>
+{versions.Aquarium_Badge}
+</ReactMarkdown>
+
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+![Aquarium addon for Ayon](assets/aquarium/aquarium-ayon-logo.png)
+
+## Introduction
+
+Welcome in the developer documentation of Aquarium addon and thanks for looking into it.
+
+This documentation is here to provide information on :
+
+- [How this code is structured](#code-structure) ?
+- [How to setup your developer environment](#developer-environment) ?
+
+:::info
+Disclaimer: We are new developers into Ayon's ecosystem.
+The following documentation might require some improvements and we may have missed steps.
+Feel free to update this file or directly reach us at [support@fatfi.sh](mailto:support@fatfi.sh)
+:::
+
+## Code structure
+
+This addon is composed by 3 main parts:
+
+- [Client](#client)
+- [Server](#server)
+  - API
+  - Frontend
+- [Services](#services)
+  - Leecher
+  - Processor
+
+### Directory
+
+#### Client
+
+Client code is used in the [Ayon Launcher](ayon-launcher), the AYON pipeline desktop application.
```

Comment:
Hello, 
The link `[Ayon Launcher](ayon-launcher)` is dead.
Was it supposed to point to the ayon-launcher repo?
```suggestion
Client code is used in the [Ayon Launcher](https://github.com/ynput/ayon-launcher), the AYON pipeline desktop application.
```

Reply:
Thanks ! I approved your suggestion. :)